projects/igniteui-angular/src/lib/progressbar/circularbar.component.spec.ts

```diff
@@ -26,6 +26,12 @@ describe('IgxCircularProgressBarComponent', () => {
         circularBar = fixture.debugElement.nativeElement;
     });
 
+    afterEach(() => {
+        // Remove elements manually from DOM, because CircularBar overrides the id the TestBed uses
+        fixture.elementRef.nativeElement.remove();
+        fixture.destroy();
+    });
```

Comment:
This is few lines above in the `beforeEach` for the exact same reason:
```ts
// For test fixture destroy
progress.id = "root1";
```
Did that stop working? Because it's used in other component's tests too.

Reply:
Oh, that's a follow up to the PR adding it. Should it be removed? Did it never work?

Reply:
Hmm I guess it has been added after I committed the changes. It appears to be working, apparently by setting id with name `root` in it. Will close this PR